ansible/hades/defaults/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+hades_version: "latest"
+hades_scheduler_version: "{{ hades_version }}"
+hades_api_version: "{{ hades_version }}"
+
+hades_node_role: "scheduler" # possible options are "scheduler" and "api"
+
+hades_api_user: "hades" # Currently not used
+hades_api_password: ""
+
+hades_api_port: 8080
+hades_api_host: "localhost"
+hades_api_certificate_fullchain_path: ""
+hades_api_certificate_key_path: ""
+
+hades_redis_host: "localhost"
+hades_redis_port: 6379
+hades_redis_password: ""
+hades_redis_version: "7.2"
+
+hades_scheduler_concurrency: 1
+
+hades_executor: "docker"
+
```

Comment:
I guess we need the other options of the ENV here too. (e.g timeout etc)

Reply:
I think we will leave them out for now and add them in a follow up.